velox/functions/sparksql/String.h

```diff
@@ -908,6 +861,61 @@ struct TranslateFunction {
     }
     result.resize(i);
   }
+
+ private:
+  FOLLY_ALWAYS_INLINE int32_t
+  getUtf8CharLength(const arg_type<Varchar>& input, int32_t offset) {
+    return std::min<int32_t>(
+        std::abs(utf8proc_char_length(input.data() + offset)),
```

Comment:
Wondering if we can apply a similar fix as that in the mask function for invalid UTF-8 character.

https://github.com/facebookincubator/velox/blob/205cbdf0792f92f6468a5d51b00be63bb39f42b7/velox/functions/sparksql/MaskFunction.h#L146-L152

Reply:
Here the required semantic is different, we can't use utf8proc_codepoint function directly.